packages/components/nodes/responsibleAI/ResponsibleAI.ts

```diff
@@ -0,0 +1,30 @@
+import { BaseLanguageModel } from 'langchain/base_language'
+import { Server } from 'socket.io'
+
+export abstract class ResponsibleAI {}
```

Comment:
@vinodkiran should we remove `ResponsibleAI` and rename to `Moderation` ? The original thinking was that we will be adding input & output moderation, but after discussion we will just add input moderation, so `ResponsibleAI` might not make sense anymore

Reply:
Agreed! I have updated the file and directory names to reflect better alignment to the functionality.